src/icons/Database/DatabaseIcon.tsx

```diff
@@ -0,0 +1,24 @@
+import { DEFAULT_HEIGHT, DEFAULT_WIDTH } from '../../constants/constants';
+import { IconProps } from '../types';
+
+export const DatabaseIcon = ({
+  width = DEFAULT_WIDTH,
+  height = DEFAULT_HEIGHT,
+  fill = 'currentColor',
```

Comment:
@amitaruyita, does it make sense that the default value for fill would be a theme reference to icon.default?